Modules/Debug/Frames.lua

```diff
@@ -320,8 +320,12 @@ function CraftSim.DEBUG.FRAMES:InitCacheTab(cacheTab)
             clearButton.clickCallback = function()
                 local cacheSV = _G[cacheName]
                 if cacheSV then
-                    wipe(cacheSV)
-                    CraftSim.DEBUG:SystemPrint(f.l("CraftSim ") .. "Cache cleared: " .. cacheName)
+                    if cacheName == "CraftSimPriceOverridesV2" then

```

Comment:
I see now where the problem is with that ^^ hm
Ill probably refactor the whole saved variable table system similar to that what I use now in PvPAssistant.
I will not do it before your PR is merged tho :D